share/completions/magento.fish

```diff
@@ -0,0 +1,720 @@
+###
+# executes fish MySQL commands
+#
+# To make this work, you'll need to create a __fish user in your
+# database, using the following mysql commands:
+#
+#   mysql> CREATE USER '__fish'@'localhost' IDENTIFIED BY 'your random pw';
+#   mysql> GRANT SELECT, SHOW ON * . * TO '__fish'@'localhost';
+#   mysql> FLUSH PRIVILEGES;
+#
+# If no credentials are given, it will simply print nothing, so further commands
+# won't show suggestions
+###
+function __fish_print_magento_mysql_command -d "Executes a MySQL command using the fish user"
+
+### IMPORTANT: ENTER FISH CREDENTIALS HERE #####
+  set -l fish_mysql_user __fish
+  set -l fish_mysql_pass fish_12479284903518
+################################################
+
+  set -l result (mysql -u{$fish_mysql_user} -p{$fish_mysql_pass} -e $argv --skip-column-names 2>&1 | grep -v "mysql:")
+  if [ (echo $result | grep ERROR) ]
+    return 1
+  else
+    for i in $result
+      echo $i
+    end
+    return 0
+  end
+end
+
+function __fish_print_magento_url_protocols -d "Shows all available URL protocols"
+  echo http://\t"HTTP"
+  echo https://\t"HTTPS"
+end
+
+function __fish_print_magento_mysql_engines
+  echo MyISAM\t"MyISAM storage engine"
+  echo InnoDB\t"Supports transactions, row-level locking, and foreign keys"
+  echo BLACKHOLE\t"/dev/null storage engine (anything you write to it disappears)"
+  echo MEMORY\t"Hash based, stored in memory, useful for temporary tables"
+  echo CSV\t"CSV storage engine"
+  echo ARCHIVE\t"Archive storage engine"
+end
+
+###
+
+###
+# Retrieves all available MySQL users
+###
+function __fish_print_magento_mysql_users -d "Shows all available MySQL users"
+    set -l users (__fish_print_magento_mysql_command 'select user from mysql.user;' | sort -u)
+
+    for i in $users
+      echo $i\t"$i"
```

Comment:
Like above, these descriptions are completely useless.

So this entire loop can just be `printf '%s\n' $users`. Alternatively, just let the output of the above command substitution go out.